engine_test.go

```diff
@@ -1036,6 +1036,14 @@ var queries = []struct {
 			{string("first row"), int64(1)},
 		},
 	},
+	{
+		`SELECT GREATEST(1, 2, 3, 4)`,
+		[]sql.Row{{int64(4)}},
+	},
+	{
+		`SELECT LEAST(1, 2, 3, 4)`,
```

Comment:
could you add some tests using strings too?

Reply:
And some tests using table columns

Reply:
@ajnavarro there are already tests with strings (the ones called "all strings", "all strings and empty", and a couple mixing strings with other types).

Reply:
@ajnavarro , @erizocosmico added tests with strings and tables to this file, PTAL.